### ISSUE TITLE
fails on blessed from DT

### ISSUE THREAD

**Issue by allahkazam** [joarwilk/typedef-converter#7]
This is an awesome tool! Thanks for making this, I've been able to convert some large but simple definitions from DT, saving me hours of work!

I ran into a problem when trying to convert blessed:
https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/master/blessed/blessed.d.ts

```
/usr/local/lib/node_modules/typedef-converter/lib/printer.js:221
  var heritage = node.heritageClauses.map(function (clause) {
                                     ^
TypeError: Cannot read property 'map' of undefined
    at printClass (/usr/local/lib/node_modules/typedef-converter/lib/printer.js:221:38)
    at Array.map (native)
    at /usr/local/lib/node_modules/typedef-converter/lib/printer.js:270:415
    at Array.map (native)
    at finalPrint (/usr/local/lib/node_modules/typedef-converter/lib/printer.js:269:6)
    at Command.program.version.arguments.option.action (/usr/local/lib/node_modules/typedef-converter/index.js:31:46)
    at Command.listener (/usr/local/lib/node_modules/typedef-converter/node_modules/commander/index.js:301:8)
    at emitOne (events.js:96:13)
    at Command.emit (events.js:188:7)
    at Command.parseArgs (/usr/local/lib/node_modules/typedef-converter/node_modules/commander/index.js:617:12)
```

**Comment by joarwilk** [comment]
I'm happy to hear that it works!

Not as happy about that definition file however, had no idea you could nest module exports. I'll look into it.

**Comment by joarwilk** [comment]
This has been resolved in version 0.1.4 🙂 The definition file will have some other problems regarding the module nesting, but thats its own issue
